package mariapiabaldoin.Giorno_3.exceptions;


public class NotFoundException extends RuntimeException {
    public NotFoundException(long id) {
        super("Il record con id " + id + " non Ã¨ stato trovato!");
    }
}